.github/workflows/assets.yml

```diff
@@ -15,177 +15,177 @@ jobs:
-      env:
-        NPROCS: 5
-      run: python setup.py bdist_wheel
-    - name: Wheel path
-      id: wheel
-      working-directory: pybuild/dist/
-      run: echo "##[set-output name=wheel;]$(ls *.whl)"
-    - uses: actions/upload-artifact@v2
-      with:
-        name: pypi-macos-py${{ matrix.python }}
-        path: pybuild/dist/${{ steps.wheel.outputs.wheel }}
-    - uses: actions/upload-release-asset@v1
-      if: ${{ github.event_name == 'release' && github.event.action == 'created' }}
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ github.event.release.upload_url }}
-        asset_path: pybuild/dist/${{ steps.wheel.outputs.wheel }}
-        asset_name: ${{ steps.wheel.outputs.wheel }}
-        asset_content_type: application/zip
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install dependencies
+        run: |
+          brew install bison flex swig libomp
+          echo "/usr/local/opt/bison/bin" >> $GITHUB_PATH
+          echo "/usr/local/opt/flex/bin" >> $GITHUB_PATH
+          python -m pip install --upgrade pip setuptools wheel
+      - name: Enable CPU compatibility mode
+        run: echo "QX_CPU_COMPATIBILITY_MODE=ON" >> $GITHUB_ENV
+      - name: Build wheel
+        env:
+          NPROCS: 5
+        run: python setup.py bdist_wheel
+      - name: Wheel path
+        id: wheel
+        working-directory: pybuild/dist/
+        run: echo "##[set-output name=wheel;]$(ls *.whl)"
+      - uses: actions/upload-artifact@v2
+        with:
+          name: pypi-macos-py${{ matrix.python }}
+          path: pybuild/dist/${{ steps.wheel.outputs.wheel }}
+      - uses: actions/upload-release-asset@v1
+        if: ${{ github.event_name == 'release' && github.event.action == 'created' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: pybuild/dist/${{ steps.wheel.outputs.wheel }}
+          asset_name: ${{ steps.wheel.outputs.wheel }}
+          asset_content_type: application/zip
 
   manylinux:
     name: PyPI wheels for Manylinux
     runs-on: ubuntu-latest
     container: quay.io/pypa/manylinux${{ matrix.manylinux }}_x86_64:latest
     env:
-      FLEX_VERSION: '2.6.4'
-      BISON_VERSION: '3.0.4-2.el7'
-      SWIG_VERSION: '3.0.12-17.el7'
-      CMAKE_VERSION: '3.13.0'
+      FLEX_VERSION: "2.6.4"
+      BISON_VERSION: "3.0.4-2.el7"
+      SWIG_VERSION: "3.0.12-17.el7"
+      CMAKE_VERSION: "3.13.0"
     strategy:
       matrix:
         manylinux:
-        - 2014
+          - 2014
         cpython_version:
-        - 'cp37-cp37m'
-        - 'cp38-cp38'
-        - 'cp39-cp39'
-        - 'cp310-cp310'
+          - "cp37-cp37m"
+          - "cp38-cp38"
+          - "cp39-cp39"
+          - "cp310-cp310"
```

Comment:
You need to add cp311-cp311 here if you want python 3.11 wheels for linux as well (you've only added them for MacOS it seems)

Reply:
Done